translate/Main.hs

```diff
@@ -14,19 +14,21 @@ import Tuura.Concept.STG
 import Tuura.Concept.STG.Simulation
 import Tuura.Concept.STG.Translation hiding (Signal)
 
+import Tuura.Plato.Options
+
 import qualified Language.Haskell.Interpreter as GHC
 import qualified Language.Haskell.Interpreter.Unsafe as GHC
 
 main :: IO ()
 main = do
-    args <- getArgs
-    if length args /= 1
-        then putStrLn "Exactly one path needed"
-        else do
-            r <- GHC.runInterpreter $ doWork (head args)
-            case r of
-               Left err -> putStrLn $ displayException err
-               Right () -> return ()
+    -- args <- getArgs
+    options <- getOptions
+    let input = optInput options
+    let paths = [input] ++ optInclude options
+    r <- GHC.runInterpreter $ doWork paths
+    case r of
+        Left err -> putStrLn $ displayException err
+        Right () -> return ()
 
```

Comment:
`either` is a useful function when working with `Either`. It either does this or that :)